### DISCUSSION TITLE
[BUG]无意间发现的一个重大BUG

### DISCUSSION THREAD

**Discussion by Couleur-Share** [1Panel-dev/1Panel#5821] [Q&A]
### 联系方式

201435454@qq.com

### 1Panel 版本

v1.10.12-beta.1

### 问题描述

因本人迁移服务器，无意间发现一个问题（可能对某些人是福利，不知道发出来会不会被骂），我在拥有专业版授权的A服务器创建快照，然后在B服务器上恢复快照，B服务器也可以获得专业版授权，此时A.B两台服务器可同时拥有专业版授权

### 重现步骤

拥有专业版授权的A服务器创建快照 -> 未拥有专业版授权的B服务器上进行恢复快照

**Comment by wanghe-fit2cloud** [comment]
服务重启时会校验设备 ID 等信息，许可证服务器也会禁止多台服务器共用许可证的情况。定期同步许可证也会禁用违规的许可证。

**Reply by Couleur-Share** [reply]
目前A.B两台服务器，都可正常更新版本，均为出现某一台许可证失效的情况（刚刚已经手动将A服务器解除绑定）

**Reply by wanghe-fit2cloud** [reply]
同步许可证后会有解绑操作。
